### DISCUSSION TITLE
Params and the simulator are showing different things

### DISCUSSION THREAD

**Discussion by rileythequeerdo** [inkstitch/inkstitch#2652] [Q&A]
When I go into the simulator everything looks fine but in params if I do satin column it'll show my text but not my design and if I do stroke the text only partially appears but the design appears, so is there a way to do satin column for the text without disabling the stroke for my design?

**Comment by tyrosinase** [comment]
When you open params, you should only have the objects of a particular type (or that you want to have be a particular type, rather) selected. Params is not a full simulator, just a single-style previewer.

If you want to change the parameters on only the satins within text, select one of the satins with ctrl-click, then right-click and choose "select same > stroke style" and it should select all the satins.

Caveat: It won't select them all if the strokes are of a different weight, unfortunately. And if you want to select only the satins within the text but others are getting selected, you'll need to do something like move the text to its own layer and hide other layers until you've made the changes.

Having "select same > stitch style" within the extension would be handy, but I'm not sure that's doable.
